File: Java/PrimeOrNot.java
import java.util.Scanner;
public class PrimeOrNot{
    public static void main(String[]args){
        Scanner sc=new Scanner(System.in);
        int num=sc.nextInt();
        int i=2;
        while(i<num){
            if(num%i==0){
                System.out.println(num+" is not a prime number");
                break;
            }
            i++;
        }
        if(i==num){
            System.out.println(i+" is a prime number");
        }
    }
}

File: Python/PrimeOrNot.py
a=2
check=0
number=int(input("Enter a number: "))
if number<=1:
    check=1

    while(a<=number/2):
     if number%a==0:
            check=1
            break
    else:
             a+=1
if check==0:print(number," is prime number")
else:print(number," is not prime number")